personio-personnel-data-api-oa3.yaml

```diff
@@ -4053,28 +4053,24 @@ components:
           format: date
           description: "Absence end date. Format: yyyy-mm-dd"
           example: '2020-01-31'
-        full_day:
```

Comment:
just to make sure this is not a breaking change - this `full_day` field was documented but never used in the actual API?

Reply:
It was actually used but redundant.

If `full_day` was false we would consider the `start_time` and `end_time` fields, if true we would not consider those fields.
Now we just check if the `start_time` and `end_time` fields are provided, if they are we use them otherwise we don't, no need for an extra boolean.

This means we cannot change it anymore, right? 😢

Reply:
Talking to the PM, there should be no customers using this API yet, so it should be fine to change it now.

Reply:
Hi @diegofaria I was not aware of the discussions happening last week. As it's confirmed through the monitoring tools, you can proceed with this PR fmpov once the behavior change MR is merged/deployed.